multiaccounts/accounts/test_helper.go

```diff
@@ -30,47 +30,51 @@ func GetWatchOnlyAccountsForTest() []*Account {
 	return []*Account{wo1, wo2, wo3}
 }
 
-func GetProfileKeypairForTest(onlyChatAndDefaultWalletAccount bool) *Keypair {
+func GetProfileKeypairForTest(includeChatAccount bool, includeDefaultWalletAccount bool, includeAdditionalAccounts bool) *Keypair {
 	kp := &Keypair{
 		KeyUID:      "0000000000000000000000000000000000000000000000000000000000000001",
 		Name:        "Profile Name",
 		Type:        KeypairTypeProfile,
 		DerivedFrom: "0x0001",
 	}
 
-	profileAccount := &Account{
-		Address:   types.Address{0x01},
-		KeyUID:    kp.KeyUID,
-		Wallet:    false,
-		Chat:      true,
-		Type:      AccountTypeGenerated,
-		Path:      "m/43'/60'/1581'/0'/0",
-		PublicKey: types.Hex2Bytes("0x000000001"),
-		Name:      "Profile Name",
-		Operable:  AccountFullyOperable,
+	if includeChatAccount {
+		profileAccount := &Account{
+			Address:   types.Address{0x01},
+			KeyUID:    kp.KeyUID,
+			Wallet:    false,
+			Chat:      true,
+			Type:      AccountTypeGenerated,
```

Comment:
just noticed this change! it was empty for mobile chat account before. before this, type="" and type= AccountTypeGenerated have different meaning, now we just assume they are the same? cc @rasom @cammellos

Reply:
@qfrank that was already discussed in one of my previous PRs. 
Yes, they are the same and means that accounts with the same `AccountTypeGenerated` type are generated from a profile keypair.

Reply:
so this is one thing mobile team need to align, could you summarise what extra need to be aligned for mobile team?

Reply:
It's not only this and also I don't know what is used in which way in mobile app, but will create a note about what's changed that somebody from mobile team can go through it, analyze changes and align mobile app with them.

Reply:
> so this is one thing mobile team need to align, could you summarise what extra need to be aligned for mobile team?

@qfrank check this comment: https://github.com/status-im/status-go/pull/3475#issuecomment-1563932045